pyxem/utils/io_utils.py

```diff
@@ -136,38 +136,86 @@ def load_hspy(filename, lazy=False, assign_to=None):
 
     return s
 
-
-def load_mib(mib_filename, reshape=True):
-    """Read a .mib file using dask and return as LazyElectronDiffraction2D.
+def load_mib(mib_path, reshape=True, flip=True, h5_stack_path=None):
+    """Read a .mib file or an h5 stack file using dask and return as a lazy pyXem / hyperspy signal.
```

Comment:
Why is this now also loading an h5_stack file? Why can't that just be done with regular hyperspy type loading?